components/SocialIcons/SocialSvgComponents.tsx

```diff
@@ -0,0 +1,66 @@
+import { SVGProps } from "react";
+
+export const FacebookIcon = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
Since we are now representing all of these svgs as components, please delete the unused svgs in `public/social`.